src/index.js

```diff
@@ -28,26 +43,27 @@ module.exports = function purescriptLoader(source, map) {
     pscIde: false,
     pscIdeColors: webpackOptions.psc === 'psa' || query.psc === 'psa',
     pscIdeArgs: {},
+    pscPackage: false,
     bundleOutput: 'output/bundle.js',
     bundleNamespace: 'PS',
     bundle: false,
     warnings: true,
     output: 'output',
     src: [
       path.join('src', '**', '*.purs'),
-      path.join('bower_components', 'purescript-*', 'src', '**', '*.purs')
+      ...depsPaths(query.pscPackage)
     ]
   }, webpackOptions, query)
 
   this.cacheable && this.cacheable()
 
   let cache = config.purescriptLoaderCache = config.purescriptLoaderCache || {
-    rebuild: false,
-    deferred: [],
-    bundleModules: [],
-    warnings: [],
-    errors: []
-  }
+      rebuild: false,
+      deferred: [],
+      bundleModules: [],
+      warnings: [],
+      errors: []
+    }
```

Comment:
Sorry to be picky, but can we put the spacing back to what it was on lines 61 to 66. It looks like just an extra space (or two) has been added.